TUTORIAL.md

```diff
@@ -0,0 +1,245 @@
+#Guide to Solving and Reviewing Rails Blogs Nested Forms
+##Overview
+This lab will teach you how to implement nested forms. In this example, we will nest tags under posts, so a user can submit a post that has many tags.
+
+###Step 1: Set up your app
+- `bin/rake db:migrate RAILS_ENV=test`
+- `bundle install`
+- `rails s` then visit `localhost:3000`
+- `rake db:seed`
+
+At this point we should have a form that will let us create a post, along with tags that we can select. Now we need a way to add a new tag, so let's add that to our form.
+
+###Step 2: Use fields_for to accept tags
```

Comment:
Jumping right into the fields stuff. Have you set the accepts_nested_attributes yet?